.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
this does not deserve here. It's not project-specific so should be in your global gitignore.